src/ru/indivio/algorithms/GrgaphWayTest.java

```diff
@@ -0,0 +1,45 @@
+package ru.indivio.algorithms;
+
+import ru.indivio.algorithms.graph.Graph;
+
+public class GrgaphWayTest {
+
+    public static void main(String[] args) {
+        WayDfs();
+    }
+
+    private static void WayDfs() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.